diff --git a/plasma/look-and-feel/com.github.vinceliuice.Monterey-dark/metadata.desktop b/plasma/look-and-feel/com.github.vinceliuice.Monterey-dark/metadata.desktop
index 9d67bb8..8d04f97 100644
--- a/plasma/look-and-feel/com.github.vinceliuice.Monterey-dark/metadata.desktop
+++ b/plasma/look-and-feel/com.github.vinceliuice.Monterey-dark/metadata.desktop
@@ -1,5 +1,5 @@
 [Desktop Entry]
-Name=Monterey-dark
+Name=KDE-Monterey-dark
 Comment=Macos Monterey theme for kde plasma
 Encoding=UTF-8
 Keywords=Desktop;Workspace;Appearance;Look and Feel;Logout;Lock;Suspend;Shutdown;Hibernate;
@@ -18,4 +18,4 @@ X-KDE-PluginInfo-Website=https://github.com/vinceliuice/Monterey-kde
 X-KDE-fallbackPackage=org.kde.breeze.desktop
 X-Plasma-MainScript=defaults
 
-X-KPackage-Dependencies=kns://colorschemes.knsrc/api.kde-look.org/1567570,kns://plasma-themes.knsrc/api.kde-look.org/1567587,kns://aurorae.knsrc/api.kde-look.org/1567564,kns://wallpaper.knsrc/api.kde-look.org/1567576,kns://sddmtheme.knsrc/api.kde-look.org/1567573,kns://icons.knsrc/api.kde-look.org/1405756,kns://xcursor.knsrc/api.kde-look.org/1411743
+X-KPackage-Dependencies=kns://colorschemes.knsrc/api.kde-look.org/1567570,kns://plasma-themes.knsrc/api.kde-look.org/2000052,kns://aurorae.knsrc/api.kde-look.org/1567564,kns://wallpaper.knsrc/api.kde-look.org/1567576,kns://sddmtheme.knsrc/api.kde-look.org/1567573,kns://icons.knsrc/api.kde-look.org/1405756,kns://xcursor.knsrc/api.kde-look.org/1411743
diff --git a/plasma/look-and-feel/com.github.vinceliuice.Monterey/metadata.desktop b/plasma/look-and-feel/com.github.vinceliuice.Monterey/metadata.desktop
index db63bca..c486d97 100644
--- a/plasma/look-and-feel/com.github.vinceliuice.Monterey/metadata.desktop
+++ b/plasma/look-and-feel/com.github.vinceliuice.Monterey/metadata.desktop
@@ -1,5 +1,5 @@
 [Desktop Entry]
-Name=Monterey
+Name=KDE-Monterey
 Comment=Macos Monterey theme for kde plasma
 Encoding=UTF-8
 Keywords=Desktop;Workspace;Appearance;Look and Feel;Logout;Lock;Suspend;Shutdown;Hibernate;
@@ -18,4 +18,4 @@ X-KDE-PluginInfo-Website=https://github.com/vinceliuice/Monterey-kde
 X-KDE-fallbackPackage=org.kde.breeze.desktop
 X-Plasma-MainScript=defaults
 
-X-KPackage-Dependencies=kns://colorschemes.knsrc/api.kde-look.org/1567570,kns://plasma-themes.knsrc/api.kde-look.org/1567562,kns://aurorae.knsrc/api.kde-look.org/1567564,kns://wallpaper.knsrc/api.kde-look.org/1567576,kns://sddmtheme.knsrc/api.kde-look.org/1567573,kns://icons.knsrc/api.kde-look.org/1405756,kns://xcursor.knsrc/api.kde-look.org/1411743
+X-KPackage-Dependencies=kns://colorschemes.knsrc/api.kde-look.org/1567570,kns://plasma-themes.knsrc/api.kde-look.org/1999932,kns://aurorae.knsrc/api.kde-look.org/1567564,kns://wallpaper.knsrc/api.kde-look.org/1567576,kns://sddmtheme.knsrc/api.kde-look.org/1567573,kns://icons.knsrc/api.kde-look.org/1405756,kns://xcursor.knsrc/api.kde-look.org/1411743
